lib/tasks/whitehall_importer.rb

```diff
@@ -120,37 +127,93 @@ def create_edition(document, translation, whitehall_edition, edition_number)
         created_at: whitehall_edition["created_at"],
       )
 
-      Edition.create!(
+      edition = Edition.create!(
         document: document,
         number: edition_number,
         revision_synced: true,
         revision: revision,
-        status: Status.new(
-          state: state(whitehall_edition),
-          revision_at_creation: revision,
-        ),
+        status: initial_status(whitehall_edition, revision),
         current: whitehall_edition["id"] == most_recent_edition["id"],
         live: live?(whitehall_edition),
         created_at: whitehall_edition["created_at"],
         updated_at: whitehall_edition["updated_at"],
         created_by_id: user_ids[first_author["whodunnit"]],
         last_edited_by_id: user_ids[last_author["whodunnit"]],
       )
+
+      set_current_status(whitehall_edition, edition) if whitehall_edition["state"] == "withdrawn"
+    end
+
+    def initial_status(whitehall_edition, revision)
+      author = if whitehall_edition["state"] == "withdrawn"
+                 published_author(whitehall_edition)
+               else
+                 current_author(whitehall_edition)
+               end
+
+      raise AbortImportError unless author
```

Comment:
What's the reason for aborting here? we probably should have some text that explains why as an argument into the exception.

I'm a little confused as the user could still not be existent on line 159